vision/standard_object/odlc_text_detection.py

```diff
@@ -0,0 +1,149 @@
+"""
+Method for reading the text on the odlc standard objects.
+"""
+
+from typing import Any, Tuple
```

Comment:
Shouldn't be importing `Tuple` in Python 3.10+. Use `tuple` instead without an import